style/hudl_sql_style_guide.md

```diff
@@ -51,7 +118,24 @@ description: A guide to writing clean, clear, and consistent SQL.
   __BAD__:
   `SELECT ksr.backings AS b`
 
-## Common Table Expressions (CTEs)
+## SQL Style Guide
+
+### General Rules
+
+* No tabs. 2 spaces per indent.
+* No trailing whitespace.
+* 80 character limit per line
+* Include links to hudl id constants or references to important links
+* Always use AS when creating aliases for columns, tables, subqueries
+* Always capitalize SQL keywords (e.g., `SELECT` or `AS`)
+* Always capitalize SQL datatypes (e.g., `VARCHAR` or `DATE`)
+* Comments should go near the top of your query, or at least near the closest `SELECT`
+* Try to only comment on things that aren't obvious about the query (e.g., why a particular ID is hardcoded, etc.)
+* All dates should be formatted as 'YYYY-MM-DD'
+* Use `!=` for the inequality operator
+
+
+### Common Table Expressions (CTEs)
 
 * Use [Common Table Expressions](http://www.postgresql.org/docs/8.4/static/queries-with.html) often, and place them at the top of code
 * CTEs should be used to represent **ALL** table/model references to make code more readable
```

Comment:
Any filters specific to one table/model should go into the CTE vs. the main query